### PR TITLE
Resolved Recently viewed products in Luma theme does not fit on scree…

### PR DESCRIPTION
**Pull request by akash2008**
### Description (*)
Using the "recently viewed products" widget on mobile will break the viewport and forcing sidescrolling.

The problem is even worse when using translated "add to cart" that make the button longer.

### Related Pull Requests

#27058 
### Fixed Issues (if relevant)

1. magento/magento2#27058 : Recently viewed products in Luma theme does not fit on screen

### Manual testing scenarios (*)

1. Add a new widget for "Recently viewed products"
2. Select Layout update "All pages" and container "Sidebar Main"
3. Visit some products in mobile view until the widget appears in mobile view

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by ptylek — CHANGES_REQUESTED** [review]
Dear @akash2008 , please fix static tests, e.g. empty lines and spaces before curly braces

**Comment by akash2008** [comment]
@ptylek  I have updated the change as per request kindly review
Let me know if anything
Thank you

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less [hunk 1]
```diff
@@ -642,6 +642,20 @@
             &:extend(.abs-no-border-top all);
         }
     }
+    
+    .sidebar {
+        .product-items {
+            .product-item-details {
+                margin: 0;
+            }
+            .product-item-info {
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:651]: Add empty line before `.product-item-info`

**Review by ptylek — CHANGES_REQUESTED** [review]
@akash2008 Thanks for fixing tests. I have checked this once again (also visually) and this also happens in Magento/blank theme. Could you also check for `block-viewed-products-grid` class and its width property for mobile and in sidebar? It seems that on mobile scenario width shouldn't split into grid and just be full-width. I checked this with other widgets like compare product or wishlist.

When width is 100% and not split like in normal grid -> maybe just scope it to sidebar? This solution might need some spacing adjustment like in other widgets
image

Your solution breaks elements of other items in sidebar
image

Please consider different solution for that issue.

**Comment by ptylek** [comment]
@akash2008 I will close this PR for now due to inactivity, please reopen if you wish to continue your work.

**Pull request closed without merging**